client/my-sites/domains/domain-management/email/add-google-apps-card.jsx

```diff
@@ -188,39 +207,30 @@ const AddGoogleAppsCard = createReactClass( {
 						</p>
 					</div>
 				</CompactCard>
-			</div>
-		);
-	},
-
-	renderAddGoogleAppsButton() {
-		const { translate } = this.props;
-
-		if ( ! config.isEnabled( 'upgrades/checkout' ) ) {
-			return null;
-		}
-
-		return (
-			<Button type="button" onClick={ this.goToAddGoogleApps }>
-				{ translate( 'Add G Suite' ) }
-			</Button>
-		);
-	},
-
-	handleLearnMoreClick() {
-		this.recordEvent( 'learnMoreClick', this.props.selectedDomainName || null );
-	},
-
-	handleAndMoreClick() {
-		this.recordEvent( 'andMoreClick', this.props.selectedDomainName || null );
-	},
-
-	goToAddGoogleApps() {
-		page(
-			domainManagementAddGoogleApps( this.props.selectedSite.slug, this.props.selectedDomainName )
+			</Fragment>
 		);
-	},
-} );
+	}
+}
+
+AddGoogleAppsCard.propTypes = {
+	products: PropTypes.object.isRequired,
+	selectedSite: PropTypes.oneOfType( [ PropTypes.object, PropTypes.bool ] ).isRequired,
+};
+
+const learnMoreClick = domainName =>
+	composeAnalytics(
+		recordTracksEvent( 'calypso_domain_management_email_learn_more_click', {
+			domain_name: domainName,
+		} ),
+		recordGoogleEvent(
+			'Domain Management',
+			'Clicked "Learn more" Google Apps link in Email',
+			'Domain Name',
+			domainName
+		)
+	);
 
 export default connect( state => ( {
 	currencyCode: getCurrentUserCurrencyCode( state ),
+	learnMoreClick,
```

Comment:
I think `learnMoreClick`  needs to be in the second argument to `connect`

```suggestion
	} ), {
	    learnMoreClick,
```

I wasn't seeing the event without it

Reply:
Fixed